lib/class-wp-rest-plugins-controller.php

```diff
@@ -37,11 +37,22 @@ public function register_routes() {
 	}
 
 	public function get_items_permissions_check( $request ) {
-
+		return true;
 	}
 
 	public function get_items( $request ) {
+		$data = array();
+
+		foreach ( get_plugins() as $obj ) {
```

Comment:
I think we need to `require_once ABSPATH . '/wp-admin/includes/plugin.php'` here, because `get_plugins()` isn't always available.